microreactors/hpmr_h2/steady/tests

```diff
@@ -0,0 +1,74 @@
+[Tests]
+  # Generate the meshes
+  [neutronics_coarse_mesh]
+    type = CheckFiles
+    input = one_twelfth_simba_core_CM.i
+    cli_args = '--mesh-only'
+    executable_pattern = 'griffin*|blue_crab*|dire_wolf*'
+    check_files = one_twelfth_simba_core_CM_in.e
+  []
+  [neutronics_fine_mesh]
+    type = CheckFiles
+    input = one_twelfth_simba_core.i
+    cli_args = '--mesh-only'
+    check_files = one_twelfth_simba_core_in.e
+    executable_pattern = 'griffin*|blue_crab*|dire_wolf*'
+    prereq = neutronics_coarse_mesh
+  []
+
+  # Syntax checks
+  [coupled_multiphysics_syntax]
+    type = RunApp
+    input = 'neutronics_eigenvalue.i'
+    check_input = True
+    executable_pattern = 'blue_crab*|dire_wolf*|fenix*'
+    prereq = neutronics_fine_mesh
+  []
+  [neutronics_only_syntax]
+    type = RunApp
+    input = 'neutronics_eigenvalue.i'
+    check_input = True
+    executable_pattern = 'griffin*|blue_crab*|dire_wolf*|fenix*'
+    cli_args = "MultiApps/active='' Transfers/active=''"
+    prereq = neutronics_fine_mesh
+  []
+  [bison_mp_syntax]
+    type = RunApp
+    input = 'thermal_ss.i'
+    check_input = True
+    executable_pattern = 'blue_crab*|dire_wolf*|bison*|fenix*'
+    prereq = neutronics_fine_mesh
+  []
+  [bison_only_syntax]
+    type = RunApp
+    input = 'thermal_ss.i'
+    check_input = True
+    executable_pattern = 'blue_crab*|dire_wolf*|bison*|fenix*'
+    cli_args = "MultiApps/active='' Transfers/active=''"
+    prereq = neutronics_fine_mesh
+  []
+  [sockeye_only_syntax]
+    type = RunApp
+    input = 'heatpipe_vapor_only.i'
+    check_input = True
+    executable_pattern = 'dire_wolf*|sockeye*|fenix*'
+  []
+
+  # Regression tests with CSV output over a minimum amount of time steps/computation
+  [sockeye_only]
+    prereq = sockeye_only_syntax
+    type = CSVDiff
+    csvdiff = 'heatpipe_vapor_only_out.csv'
+    input = 'heatpipe_vapor_only.i'
+    executable_pattern = 'dire_wolf*|sockeye*|fenix*'
+  []
+
```

Comment:
can we test the solo-griffin and solo-bison inputs?

Reply:
tests added